### PR TITLE
feat: add component deployment requirements

### PR DESCRIPTION
**Pull request by 0xff-dev**
feat: u4a deployment tutorial
Fix: https://github.com/kubebb/components/issues/41

### CODE REVIEW

File: examples/u4a-deploy-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+spec:
+  url: http://1.1.1.1:9999/def
+  pullStategy:
+    intervalSeconds: 120
+    retry: 5
+```
+
+## 2 cluster-comonent
+
+直接创建 `componentplan`, clustr-component 会部署 `ingress-nginx` 需要选择部署节点名称，默认是`kind-worker`
```
Comment (Abirdcfly) [inline examples/u4a-deploy-tutorial.md:33]: clustr -> cluster?

File: examples/u4a-deploy-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+创建包含 `u4a-component` 相关组件的 Repository
+
+```yaml
+apiVersion: core.kubebb.k8s.com.cn/v1alpha1
+kind: Repository
+metadata:
+  name: kubebb
+  namespace: kubebb-system
+spec:
+  url: http://1.1.1.1:9999/def
```
Comment (Abirdcfly) [inline examples/u4a-deploy-tutorial.md:25]: 这里1.1.1.1，是否改成默认的github chart repo？这样，即使直接copy配置也不会错

File: examples/u4a-deploy-tutorial.md [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+```shell
+helm repo add kubebb https://kubebb.github.io/components
+kubectl create ns kubebb-system
+helm -nkubebb-system install kubebb/core
+
+NAME                     READY   STATUS    RESTARTS   AGE
+kubebb-755784d5f-k8jhh   1/1     Running   0          2m14s
+```
+
+## 1.1 create repository
```
Comment (bjwswang) [inline examples/u4a-deploy-tutorial.md:14]: should use `repo kubebb` directly

File: examples/u4a-deploy-tutorial.md [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+fabric-minio-0                                           1/1     Running   0          5h53m
+fabric-minio-1                                           1/1     Running   0          5h53m
+fabric-minio-2                                           1/1     Running   0          5h53m
+fabric-tekton-tekton-operator-766f8795ff-4g55n           2/2     Running   0          19s
+fabric-tekton-tekton-operator-webhook-64c8f79546-jrnvd   1/1     Running   0          19s
+```
+
+## 6 Fabric-Opertor
+
+Fabric-operator 需要的鸡哥关键参数
```
Comment (Abirdcfly) [inline examples/u4a-deploy-tutorial.md:158]: 鸡哥 -> 几个？

File: examples/u4a-deploy-tutorial.md [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+```yaml
+apiVersion: core.kubebb.k8s.com.cn/v1alpha1
+kind: ComponentPlan
+metadata:
+  name: cluster-compnent
+  namespace: u4a-system
+spec:
+  approved: true
+  name: cluster-component
+  version: 0.1.0
```
Comment (bjwswang) [inline examples/u4a-deploy-tutorial.md:46]: use https://github.com/kubebb/core/blob/main/tests/kind-config-3nodes.yaml this kubrnetes cluster as the sample cluster. And use node `kubebb-core-worker` as the deployedHost for ingress

**Review by Abirdcfly — APPROVED** [review]

**Pull request merged**